test_run_poly_example_with_multiple_design_matrix_instances fails

```
        with warnings.catch_warnings(record=True) as all_warnings:
            run_cli(
                ENSEMBLE_EXPERIMENT_MODE,
                "--disable-monitoring",
                "poly.ert",
                "--experiment-name",
                "test-experiment",
            )
        warning_messages = [str(warning.message) for warning in all_warnings]
        # there are two warnings, where one is about NUM_REALIZATIONS begin greater than
        # the number of realizations in the design matrix
>       assert len(warning_messages) == 2
E       AssertionError: assert 18 == 2
E        +  where 18 = len(['Use of legacy_ertscript_workflow is deprecated. Please see documentation for how to use ertscript_workflow instead',...e of legacy_ertscript_workflow is deprecated. Please see documentation for how to use ertscript_workflow instead', ...])

/tmp/f_scout_ci/actions-runner-02/_temp/source_root/tests/ert/ui_tests/cli/analysis/test_design_matrix.py:299: AssertionError
```

Onprem
https://github.com/equinor/komodo-releases/actions/runs/23125640065/job/67171922947
Azure
https://github.com/equinor/komodo-releases/actions/runs/23126241228/job/67170661244

--- a/tests/ert/ui_tests/cli/analysis/test_design_matrix.py
+++ b/tests/ert/ui_tests/cli/analysis/test_design_matrix.py
@@ -293,7 +293,13 @@ def test_run_poly_example_with_multiple_design_matrix_instances():
             "--experiment-name",
             "test-experiment",
         )
-    warning_messages = [str(warning.message) for warning in all_warnings]
+    warning_messages = [
+        str(warning.message)
+        for warning in all_warnings
+        if not str(warning.message).startswith(
+            "Use of legacy_ertscript_workflow is deprecated"
+        )
+    ]
     # there are two warnings, where one is about NUM_REALIZATIONS begin greater than
     # the number of realizations in the design matrix
     assert len(warning_messages) == 2

